monix-eval/shared/src/main/scala/monix/eval/internal/TaskConversions.scala

```diff
@@ -107,6 +110,49 @@ private[eval] object TaskConversions {
       case _ => fromConcurrentEffect0(fa)
     }
 
+  /**
+    * Implementation for `Task.fromReactivePublisher`.
+    */
+  def fromReactivePublisher[A](source: Publisher[A]): Task[Option[A]] =
+    Task.cancelable0 { (scheduler, cb) =>
+      val sub = SingleAssignSubscription()
+
+      source.subscribe(new Subscriber[A] {
+        private[this] var isActive = true
+
+        def onSubscribe(s: org.reactivestreams.Subscription): Unit = {
```

Comment:
nitpick: can you write `org.reactivestream` in imports instead of writing it here.